app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        if 1 <= comfort_class <= 7:
+            self.comfort_class = comfort_class
+
+        if 1 <= clean_mark <= 10:
+            self.clean_mark = clean_mark
```

Comment:
what if value is incorrect? it means class will not have these attributes at all